### PR TITLE
fix ellegirl.ru #33353

### PR DESCRIPTION
**Pull request by zloyden**
https://github.com/AdguardTeam/AdguardFilters/issues/33353

### CODE REVIEW

**Comment by AdamWr** [comment]
In my opinion rules looks okay, but if I'm not wrong, there are also ad leftovers on this website.
<details><summary>Screenshot</summary>

image

</details>

I think that this rule should fixes these ad leftovers:
`ellegirl.ru##.creative-wrapper`

Could you please check it too?

**Review by Alex-302 — CHANGES_REQUESTED** [review]
please block ad leftovers

File: SocialFilter/sections/specific.txt [hunk 1]
```diff
@@ -61,6 +61,7 @@ vgorode.ua##.social_plugins
 fakty.com.ua##.like_box > li:not(.f_right)
 afmen.online##.wrap-socials_webnavoz
 ipress.ua##.yashare
+ellegirl.ru##.social-container
```
Comment (Alex-302) [inline SocialFilter/sections/specific.txt]: just links to blogs. Usually we don't block them

**Review by Alex-302 — CHANGES_REQUESTED** [review]

File: SocialFilter/sections/specific.txt (hunk 1 above)
Comment (BlazDT) [reply SocialFilter/sections/specific.txt]: They are hiding the widgets which show e. g. their posts, not a link to their e. g. facebook site.

Comment (Alex-302) [reply SocialFilter/sections/specific.txt]: I see. @zloyden please also block `http://platform.twitter.com/widgets.js` (add domain to existing rule)

**Pull request merged**